index.js

```diff
@@ -118,7 +118,7 @@ Blake2b.SUPPORTED = typeof WebAssembly !== 'undefined'
 Blake2b.ready = function (cb) {
   if (!cb) cb = noop
   if (!wasmPromise) return cb(new Error('WebAssembly not supported'))
-  return wasmPromise.then(() => cb())
+  return wasmPromise.then(cb, cb)
```

Comment:
i think you need .then(() => cb(), cb) since wasmPromise returns a value unless I'm mistaken.

Reply:
It's a void promise:

```js
var wasmPromise = typeof WebAssembly !== "undefined" && require('./blake2b')().then(mod => {
  wasm = mod
})
```

The resolved module is assigned to `was`.

Reply:
Better safe than sorry though 😄